tasks/populate-release-notes-images/populate-release-notes-images.yaml

```diff
@@ -45,7 +45,7 @@ spec:
         do
             component=$(jq -c --argjson i "$i" '.components[$i]' "${SNAPSHOT_FILE}")
             name=$(jq -r '.name' <<< $component)
-            repo=$(jq -r '.repository' <<< $component)
+            repo=$(jq -r '.["rh-registry-repo"]' <<< $component)
             deliveryRepo=$(translate-delivery-repo $repo | jq -r '.[] | select(.repo=="redhat.io") | .url')
```

Comment:
Shouldn't we now have both urls available to us in the component without needing to call the translate script?